solidity/ecdsa/test/WalletRegistry.Inactivity.test.ts

```diff
@@ -176,7 +189,8 @@ describe("WalletRegistry - Inactivity", () => {
                         subsequentInactiveMembersIndices,
                         groupThreshold,
                         (signatures) => signatures,
-                        (signingMembersIndices) => signingMembersIndices
+                        (signingMembersIndices) => signingMembersIndices,
+                        1_210_000
```

Comment:
It is not entirely clear to me why we assert the amount of gas used. Do we want to make sure this does not get too expensive? What about other functions? Or maybe there is some other reason?

Reply:
We use them to track gas usage changes, to bring our attention in case the gas usage changes significantly. It is also helpful when estimating gas offsets for reimbursements configuration.